library/src/test/java/com/novoda/downloadmanager/LiteDownloadManagerTest.java

```diff
@@ -104,6 +104,15 @@ public void triggersDownloadOfBatches_whenSubmittingAllStoredDownloads() {
             inOrder.verify(downloadManagerDownloader).download(additionalDownloadBatch, downloadingBatches);
         }
 
+        @Test
+        public void doesNotPutBatchesIntoDownloadsMap_whenSubmittingAllStoredDownloads() {
```

Comment:
I find this one very misleading. It doesn't put it into the map in this class true, *but* it does do it when calling `download` on the downloader, which is called from `submitAllStoredDownloads`.